content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -285,7 +285,7 @@ This file will be used by `kubeadm init` and `kubeadm join` to source extra
 user defined arguments for the kubelet.
 
 Please mind, that you **only** have to do that if the cgroup driver of your CRI
-is not `cgroupfs`, because that is the default value in the kubelet already.
+is not `systemd`, because that is the default value in the kubelet already.
```

Comment:
/hold

i'm not aware of such a change in the kubelet.
AFAIK, the kubelet default driver is still `cgroupfs`.

/sig cluster-lifecycle

Reply:
Yes, I'm sorry I didn't read the [Reference](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/#options) part of kubelet.

> --cgroup-driver string
> --
>   | Driver that the kubelet uses to manipulate cgroups on the host. Possible values: 'cgroupfs', > 'systemd' (default "cgroupfs")

From next time I'll look for if an issue already solve my question or create one istead.

/close